build/paths.js

```diff
@@ -36,4 +38,21 @@ paths.js.root             = path.join(paths.root,'src','js')
 paths.rust                = {}
 paths.rust.root           = path.join(paths.root,'src','rust')
 
+function project_manager_path(root) {
+    let base_path = path.join(root, 'bin')
+    const target_platform = os.platform()
+    switch (target_platform) {
+        case 'linux':
+            return path.join(base_path, 'project-manager')
+        case 'darwin':
+            return path.join(base_path, 'project-manager')
+        case 'win32':
+            return path.join(base_path, 'project-manager.exe')
+        default:
```

Comment:
1. Try not using `returns` - this code is harder to read
2. Refactor common parts out (you are repeating the project anager name 3 times now, it would be mmuch better to compute the extension instead).